python/build-wheel-manylinux1.sh

```diff
@@ -13,7 +13,7 @@ rm -f /usr/bin/python2
 ln -s /opt/python/cp27-cp27m/bin/python2 /usr/bin/python2
 
 mkdir .whl
-for PYTHON in cp27-cp27mu cp34-cp34m cp35-cp35m cp36-cp36m; do
```

Comment:
Whether this works or not just depends on whether the base image already has Python 3.7 on it. The base image is the one used for building Arrow wheels.